index.html

```diff
@@ -4600,14 +4649,16 @@ <h2 id="AudioBuffer">
           <dd>
             <p>
               The sample-rate for the PCM audio data in samples per second.
+              This MUST return the value of <var>[[sample rate]]</var>.
```

Comment:
In theory, that's the style we should adopt, since we're describing something beneath the WebIDL layer. In practice, Anne told me it was OK (when really obvious) to simply refer to the attributes for now (like we do for other interfaces), since we're in a rush.

I figured that, because I was kind of rewriting stuff, that I would write it in the preferred style, and also because `duration` is not directly returned from another attribute.